packages/project-starter/src/plugin.ts

```diff
@@ -183,13 +183,54 @@ const plugin: Plugin = {
   },
   models: {
     [ModelType.TEXT_SMALL]: async (
-      _runtime,
-      { prompt, stopSequences = [] }: GenerateTextParams
+      runtime,
+      { prompt, stopSequences = [], temperature = 0.7, maxTokens = 1024 }: GenerateTextParams
     ) => {
-      return 'Never gonna give you up, never gonna let you down, never gonna run around and desert you...';
+      // Implement real OpenAI fetch call without fallbacks
```

Comment:
What's the reason for doing this? the open ai key may not always be present so we can't do this